src/System.Numerics.Vectors/tests/Vector3Tests.cs

```diff
@@ -61,7 +61,6 @@ public void Vector3GetHashCodeTest()
             Assert.False(v4.GetHashCode() == v8.GetHashCode());
             Assert.False(v7.GetHashCode() == v6.GetHashCode());
             Assert.False(v8.GetHashCode() == v6.GetHashCode());
-            Assert.True(v8.GetHashCode() == v7.GetHashCode());
```

Comment:
Rather than no longer asserting the comparison is true, shouldn't the test now assert that it's not equal?  And I don't mean using `==` here - this should be using `Assert.Equal(...)`/`Assert.NotEqual(...)`.